bin/convert-diamond-to-json.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python
+
+import argparse
+import bz2file
+import sys
+
+from dark.blast.conversion import DiamondTabularFormatReader
+
+
+if __name__ == '__main__':
+
+    parser = argparse.ArgumentParser(
+        description='Convert a DIAMOND tabular output file to JSON.',
+        epilog=('Give a DIAMOND tabular file and convert it to JSON. '
+                'Optionally compress the JSON output. Make sure you have the '
+                'right output flags set: The flags are: qseqid, sseqid, '
```

Comment:
I think you should give the exact command line fragment that it's expected diamond has been run with.